src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -38,6 +38,33 @@ public class DriveSubsystem extends SubsystemBase {
   // ENCODER DECLARATION
   private final RelativeEncoder m_leftEncoder = m_leftLeaderMotor.getEncoder();
   private final RelativeEncoder m_rightEncoder = m_rightLeaderMotor.getEncoder();
+  
+
+  private void setPidGains(){
+    leftSpeedPID.setP(0.0); //Integral, derivative, and proportional gains for the PID controller
+    leftSpeedPID.setD(0.0);
+    leftSpeedPID.setI(0.0);
+  
+    rightSpeedPID.setP(0.0);
+    rightSpeedPID.setD(0.0);
+    rightSpeedPID.setI(0.0);
+  }
+
+  double DriveDistance=0.0; //Allows setDistanceToDrive to accept an argument
```

Comment:
Is there a reason why we're storing this `DriveDistance` variable outside of the `setDistanceToDrive` method?

Reply:
Ok i got rid of that. double DriveDistance was already defined within the setdistancetodrive method